tests/test_arcaflow_plugin_horreum_client.py

```diff
@@ -9,6 +9,7 @@
 
 
 plugin_input = horreum_client_plugin.InputParams(
+    tls_verify=False,
     horreum_url="https://horreum.foo.bar.com",
```

Comment:
What if it doesn't listen to http but you don't have the CA bundle? As we've been through with Pbench??? Being able to disable verification is great for early debugging: for "real" deployment, though, we should be identifying the proper trusted CA bundle if it's not installed. (And, as we've seen, the use of `certifi`, which doesn't include the Red Hat IT CA, can be more a problem than a solution if you're using the PyPi `requests` instead of the Fedora RPM `requests`...)

Reply:
Those are all good points (see my earlier comment about welcoming Dustin to the slippery slope...), but, in this specific case, I'm reasonably certain that the `Responses` package will intercept the request long before there is any question about actually establishing a network connection, so there won't be any need for a CA bundle...in fact, the setting of `tls_verify` probably doesn't even matter, here.  😛

Reply:
Ah, yes; this is a test. You're right -- it's unlikely that the mocks care, and wouldn't have access to the real certs anyway. Your comment was specific to the context, but mine was a lot more generic and arguably doesn't really belong here. Ah, well.

Reply:
In German, I would say in this case, "Es ist egal." From the standpoint of the test code, it doesn't really make a difference if this is here or not, except the tiny argument that this tests changing the argument away from the default.